components/Heading1.tsx

```diff
@@ -0,0 +1,10 @@
+import { TypographyProps } from '@mui/material';
```

Comment:
```suggestion
import { TypographyProps } from '@mui/material/Typography';
```

czy nie powinno być tak?

Reply:
Poprawione